src/haiway/files/types.py

```
@@ -0,0 +1,111 @@
+from contextlib import AbstractAsyncContextManager
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)